doc/source/whatsnew/v0.19.0.txt

```diff
@@ -719,6 +722,21 @@ New Behavior:
    pd.NaT + 1
    pd.NaT - 1
 
+``PeriodIndex.values`` now returns array of ``Period`` object
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+``.values`` is changed to return array of ``Period`` object, rather than array
+of ``int64`` (:issue:`13988`)
+
+.. code-block:: ipython
+   In [6]: pi = pd.PeriodIndex(['2011-01', '2011-02'], freq='M')
+   In [7]: pi.values
+   array([492, 493])
+
+.. ipython:: python
+
+   pi = pd.PeriodIndex(['2011-01', '2011-02'], freq='M')
+   pi.values
```

Comment:
@sinhrks is it correct to say that to get the integer values as before (if you for some reason were using it), you cas access it through `.asi8` ?